src/core/server/bootstrap.ts

```diff
@@ -83,6 +83,9 @@ export async function bootstrap({ configs, cliArgs, applyConfigOverrides }: Boot
   try {
     await root.setup();
     await root.start();
+
+    // notify parent process know when we are ready for dev mode.
+    process.send!(['SERVER_LISTENING']);
```

Comment:
`process.send` is only available when the process was started with `childProcess.fork`, so we need to check for it to exist and not use `!`

Reply:
TIL, I thought it was a stupid type definition. Thanks